pkg/url/url.go

```diff
@@ -880,13 +884,34 @@ func Push(client *occlient.Client, kClient *kclient.Client, parameters PushParam
 		urls := parameters.EnvURLS
 		for _, url := range urls {
 			if url.Kind != envinfo.DOCKER {
+				if parameters.EndpointMap == nil {
+					klog.V(4).Infof("No Endpoint entry defined in devfile.")
+					return nil
+				}
+				exist := false
+				var endpoint parsercommon.Endpoint
+				if parameters.EndpointMap != nil {
```

Comment:
We don't need this check. The condition is checked above.

Reply:
updated